README.md

```diff
@@ -3,8 +3,60 @@
+
+We are currently at work on Consonance 2.0 which supports anything from Dockstore and allows users to submit a variety of jobs intended for a variety of instance types.
+
+The latest unstable releases on develop support running tools/workflows from  Dockstore.
+
+Consonance 1.0 is currently in maintenance mode and has also been integrated into a simplified workflow launcher Pancancer Workflow Launcher for AWS for external users and a more flexible but less well documented Pancancer Workflow Launcher for users internal to OICR.
+
 The latest stable releases on master support the pancancer project https://icgc.org/working-pancancer-data-aws
 
-The latest unstable releases on develop support https://dockstore.org/
+See the Consonance [wiki](https://github.com/Consonance/consonance/wiki) for more information on this project.
+
+## Building
+
+The build uses maven (3.2.3, look at using [MVNVM](http://mvnvm.org/)), just run:
+
+    mvn clean install
+
+ To avoid tests (probably a bad idea!):
+
+    mvn -Dmaven.test.skip=true clean install
+
+## Releasing
+
+See the [developer page](https://github.com/Consonance/consonance/wiki/developers) on our wiki.
+
+## Installation
+
+See the container-admin [README](container-admin/README.md) for information on quickly setting up Consonance via Docker-compose and an interactive bootstrap configuration script.
+
+## Using
+
+See the [quickstart guide](https://github.com/Consonance/consonance/wiki/quickstart) and the [non-quickstart guide](https://github.com/Consonance/consonance/wiki/non-quickstart-users) on our wiki.
+
+## TODO
+
+Consonance is a work in progress, there are many features and bugs that need to be fleshed out. Here are the most imporant issues.
 
-See the [wiki](https://github.com/Consonance/consonance/wiki) for more information
+* can the indivdual images be hosted on Quay.io so the install_boostrap doesn't need to spend time building them?
+* there are a ton of settings that the bootstrapper should expose to end users.  Regions, AMI, etc
+* need to setup the previous job hash and expose that through as a callable parameter to the consonance command line
+* should generate new token for admin on each deployment, should also use better passwords assigned at build time for rabbitmq and postgres
+* it seems like the instance type is hard-coded for Youxia yet it's a param for Consonance.  It really should be a param otherwise a given deployment will only work for a particular AMI/instance type.
+* the initial job submitted seem to go into the START state and never get processed
+* re-enqueing work for lost jobs ultimately fails
```

Comment:
If this refers to the code at https://github.com/Consonance/consonance/pull/133/files#diff-76ff958696cc5e026371523b195ec791R364 does this mean this should be reverted?